### DISCUSSION TITLE
关于组件文档的生成思路的请教

### DISCUSSION THREAD

**Discussion by longshihui** [tusen-ai/naive-ui#6208] [General]
作者你好，大家好，最近本人也在团队内制作前端组件物料，关于组件文档这块想请教一下思路。
为什么naive-ui对于单个组件的文档，例如Props或Slots或者是事件，使用的是手写Markdown，而不是使用其他工具（例如vue-docgen-api）生成组件元数据，再基于元数据通过代码自动生成Markdown table的思路？是出于什么考虑呢？
如果大家还有其他思路，也可以说一下，非常感谢

**Comment by jahnli** [comment]
可能是方便自定义
